.goreleaser.yml

```diff
@@ -0,0 +1,43 @@
+# Check the documentation at https://goreleaser.com for more options
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - id: notation-azure-kv
+    dir: cmd/notation-akv
+    binary: notation-azure-kv
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - darwin
+      - linux
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    ignore:
+      - goos: windows
+        goarch: arm64
+    ldflags:
+      - -s -w -X {{.ModulePath}}/internal/version.Version={{.Version}} -X {{.ModulePath}}/internal/version.BuildMetadata=
```

Comment:
Where is the `ModulePath` defined?

Reply:
xref: https://goreleaser.com/customization/templates/

.ModulePath  - the go module path, as reported by `go list -m`

Reply:
This is from the goreleaser template https://goreleaser.com/customization/templates/